Clients/src/presentation/components/PluginWidgets/templates/EmptyPageTemplate.tsx

```
@@ -0,0 +1,1047 @@
+            borderRadius: 1,
+            minHeight: 300,
+            "& img": { maxWidth: "100%" },
+            "& table": { width: "100%", borderCollapse: "collapse" },
+            "& th, & td": {
+              border: "1px solid #e5e7eb",
+              padding: "8px 12px",
+              textAlign: "left",
+            },
+            "& th": { backgroundColor: "#f9fafb", fontWeight: 600 },
+          }}
+          dangerouslySetInnerHTML={{ __html: content.html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)